code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -292,6 +292,9 @@ var/global/list/image/splatter_cache=list()
 /obj/effect/decal/cleanable/mucus/proc/set_dry()
 	dry = TRUE
 
+/obj/effect/decal/cleanable/mucus/update_icon()
+	. = ..()
```

Comment:
Ok but why

Reply:
cause you have to call your parents sometimes

Reply:
I just called them, they're doing fine. Why do you need to call parents implicitly if they're already being called without this piece of code???